tmt/cli.py

```diff
@@ -957,19 +957,19 @@ def plans_create(
 @plans.command(name='export')
 @click.pass_context
 @filter_options_long
-@click.option(
+@option(
     '-h', '--how', metavar='METHOD', default=_plan_export_default, show_default=True,
     help='Output format.',
     type=click.Choice(choices=_plan_export_formats))
-@click.option(
+@option(
     '--format', metavar='FORMAT', default=_plan_export_default, show_default=True,
     help='Output format. Deprecated, use --how instead.',
```

Comment:
This is also deprecated.

Reply:
As mentioned above, I'd rather have a separate PR for this transition.